Common.props

```diff
@@ -1,11 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Global.props'))\Global.props" />
-  <Import Project=".\GlobalStaticVersion.props" />
```

Comment:
Yes. I didn't dig into the exact solution details. So if you think your solution better - can you please as a minimum create an issue in `dotnet` repo